notebooks/00020_fix_positions_hg38_missense_pathogenic.ipynb

```diff
@@ -1224,7 +1224,11 @@
    "id": "162968cb-3f83-4aab-8c76-e557e5a21cc0",
```

Comment:
<p>@dianagroch can you please check this?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/zdenkas/human-tafazzin-variants-database/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Reply:
<p>if the correct is c.590G&gt;T (according to paper) than the correct position should be:</p><p>hg38 <a href="https://genome.ucsc.edu/cgi-bin/hgTracks?db=hg38&amp;position=chrX:154420037-154420040" target="_blank">chrX:154420038</a> p.G197V</p><p> </p><p>question is if the ref. 23 is correct in this case (see row 256 for same variant, in ref 22 patient 4 - different mutation G877T )</p><p>this should be checked with submitters</p>